src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -97,28 +108,85 @@ trait LocalFileSystemOperations {
    * Original input path: /could/be/anywhere/input.bam
    * Host inputs path: $PWD/cromwell-executions/some-workflow-name/0f00-ba4/workflow-inputs/input.bam
    */
-  private def stageWorkflowInputs(descriptor: WorkflowDescriptor): HostInputs = {
+  private def stageWorkflowInputs(descriptor: WorkflowDescriptor): Try[HostInputs] = {
     val hostInputsPath = Paths.get(LocalBackend.hostExecutionPath(descriptor).toFile.getAbsolutePath, "workflow-inputs")
-    descriptor.actualInputs map {case(name, value) => name -> stageWdlValue(value, hostInputsPath)}
+    val attemptedStagedInputs = descriptor.actualInputs map { case(name, value) =>
+      val call = descriptor.namespace.resolve(name.split("\\.").dropRight(1).mkString(".")) match {
+        case Some(c: Call) => Some(c)
+        case _ => None
+      }
+      name -> stageWdlValue(call, value, hostInputsPath)
+    }
+    attemptedStagedInputs collect { case (k: String, f: Failure[_]) => k -> f } match {
+      case m: Map[FullyQualifiedName, Failure[_]] if m.nonEmpty =>
+        val errors = m.values map { _.exception.getMessage } mkString "\n"
+        Failure(new UnsupportedOperationException(s"Failures during localization:\n\n$errors"))
+      case _ =>
+        Success(attemptedStagedInputs map { case (k, v) => k -> v.get})
```

Comment:
this looks like it could be simpler, mb something like 
```scala
attemptedStagedInputs.values collect { case f: Failure[_] => f } match {
      case Nil => Success(attemptedStagedInputs mapValues { _.get })
      case failedInputs =>
        val errors = failedInputs map { _.exception.getMessage } mkString "\n"
        Failure(new UnsupportedOperationException(s"Failures during localization:\n\n$errors"))
    }
```